src/Http/Session.php

```diff
@@ -41,9 +41,9 @@ class Session
     /**
      * The Session handler instance used as an engine for persisting the session data.
      *
-     * @var \SessionHandlerInterface
+     * @var \SessionHandlerInterface|null
      */
-    protected SessionHandlerInterface $_engine;
+    protected ?SessionHandlerInterface $_engine = null;
```

Comment:
Is this because the session is always created somewhere even if the app doesn't support sessions?

Reply:
No, the default session handling by PHP  doesn't require creating any cusom handler.